src/Constants.ts

```diff
@@ -36,6 +36,12 @@ export enum RuleType {
 	DFA = "dfa"
 }
 
+export enum MissingOptionsBehavior {
+	HALT = 'halt',
+	WARN = 'warn',
+	SKIP = 'skip'
```

Comment:
These were the first names I thought of. I'm prepared to change them if people can come up with better ones.
I went with an enum instead of a boolean to allow for more diversity of behavior. Most intriguing is the possibility of adding a "prompt" option at some point in the future.

Reply:
Looks good to me 👍